xml/XSLT/panelfile-2-9-6.xsl

```diff
@@ -150,15 +162,15 @@ This page was produced by <a href="http://jmri.org">JMRI</a>.
     <th>Out Sensors</th>
     <th>Comment</th>
     </tr>
-    <!-- index through individal route elements -->
+    <!-- index through individual route elements -->
     <xsl:apply-templates/>
     </table>
 </xsl:template>
 
 <xsl:template match="route">
 <tr>
-<td><xsl:value-of select="@systemName"/></td>
```

Comment:
I think the ```@``` symbols need to stay--IIRC that symbol indicates the start of a replacement value. This is true throughout the file, although this is the only place I'm asking about it.